packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -165,14 +168,15 @@ export default connectStore(() => {
           </BlockLink>
         </Col>
       </Row>
-      {!vm.xenTools && vm.power_state === 'Running' && (
-        <Row className='text-xs-center'>
-          <Col>
-            <Icon icon='error' />
-            <em> {_('noToolsDetected')}.</em>
-          </Col>
-        </Row>
-      )}
+      {!vm.xenTools &&
+        vm.power_state === 'Running' && (
+          <Row className='text-xs-center'>
+            <Col>
+              <Icon icon='error' />
+              <em> {_('noToolsDetected')}.</em>
+            </Col>
+          </Row>
+        )}
```

Comment:
Your deps are not up to date which introduced this formatting change, run `yarn` and fix this change please.